include/mbedtls/mbedtls_config.h

```diff
@@ -1446,13 +1446,13 @@
  *
  * \note To genuinely benefit from the smaller code size of p256-m, make
  *       sure that you do not enable any ECC-related option that requires
- *       the built-in implementation of elliptic curve arithmetic. This
- *       means enabling #MBEDTLS_PSA_CRYPTO_C, #MBEDTLS_PSA_CRYPTO_CONFIG,
- *       #PSA_WANT_ECC_SECP_R1_256 and #MBEDTLS_PSA_P256M_DRIVER_ENABLED,
- *       plus any of the `PSA_WANT_ALG_xxx` and `PSA_WANT_KEY_TYPE_xxx`
- *       options listed above, and not enabling other ECC-related options
- *       through `PSA_WANT_xxx` or `MBEDTLS_xxx` (in particular, not
- *       enabling other curves or EC-JPAKE).
+ *       the built-in implementation of elliptic curve arithmetic. Make sure
+ *       #PSA_WANT_ALG_DETERMINISTIC_ECDSA, #PSA_WANT_ALG_JPAKE and
+ *       #PSA_WANT_KEY_TYPE_ECC_KEY_PAIR_DERIVE, and curves other than
+ *       SECP256R1 are disabled as they are not supported by this driver.
```

Comment:
Non-blocker: I'm not totally happy with the wording here: "Make sure ... are disabled, as they are not supported by this driver" - from a non-careful read, it sounds a bit like those features won't be present, or maybe won't work, if you enable MBEDTLS_PSA_P256M_DRIVER_ENABLED.

Maybe something like:

```
 * \note To benefit from the smaller code size of p256-m, make
 *       sure that you do not enable any ECC-related option not supported by
 *       p256-m. This would cause the built-in ECC implementation to be built
 *       as well, in order to provide the required option.
 *       Make sure #PSA_WANT_ALG_DETERMINISTIC_ECDSA, #PSA_WANT_ALG_JPAKE and
 *       #PSA_WANT_KEY_TYPE_ECC_KEY_PAIR_DERIVE, and curves other than
 *       SECP256R1 are disabled as they are not supported by this driver.
 *       Also, avoid defining #MBEDTLS_PK_PARSE_EC_COMPRESSED or
 *       #MBEDTLS_PK_PARSE_EC_EXTENDED as those currently require a subset of
 *       the built-in ECC implementation, see docs/driver-only-builts.md.
```

(but I'm not hugely attached to my suggested alternative - maybe you can find some clearer way to express this).

Reply:
This paragraph opens with “To benefit from the smaller code size of p256-m”. If you enable those features, they'll work, but you won't benefit from the smaller code size since the built-in implementation will also be built.

Reply:
Yes, I agree it's correct and won't confuse a careful reader - but I think saying "Make sure X, Y are disabled as they are not supported by this driver" could be misunderstood. Not a blocker though if Manuel is also happy with the current wording.